mediaapi/routing/routing.go

```diff
@@ -83,15 +85,21 @@ func makeDownloadAPI(
 	activeRemoteRequests *types.ActiveRemoteRequests,
 	activeThumbnailGeneration *types.ActiveThumbnailGeneration,
 ) http.HandlerFunc {
-	return prometheus.InstrumentHandler(name, http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+	counterVec := promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: name,
+			Help: "Total number of media_api requests for either thumbnails or full downloads",
+		},
+		[]string{"code"},
+	)
+	httpHandler := func(w http.ResponseWriter, req *http.Request) {
 		req = util.RequestWithLogging(req)
 
 		// Set common headers returned regardless of the outcome of the request
 		util.SetCORSHeaders(w)
 		// Content-Type will be overridden in case of returning file data, else we respond with JSON-formatted errors
 		w.Header().Set("Content-Type", "application/json")
-
-		vars, _ := common.URLDecodeMapValues(mux.Vars(req))
+		vars := mux.Vars(req)
```

Comment:
Why drop the call to `common.URLDecodeMapValues`? It sounds like something we'd like to keep here.

Reply:
Ah, yes. This is because this was changed after I made the original fix which I just copy-pasted for this PR.